src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -55,6 +57,7 @@ internal ArrayList(bool trash)
         // Constructs a ArrayList. The list is initially empty and has a capacity
         // of zero. Upon adding the first element to the list the capacity is
         // increased to _defaultCapacity, and then increased in multiples of two as required.
+        /// <summary>Initializes a new instance of the <see cref="T:System.Collections.ArrayList" /> class that is empty and has the default initial capacity.</summary>
```

Comment:
@cartermp thanks for chatting this morning; as discussed let's start by finalizing the dev guidance docs (rationalizing // vs ///, line lengths, etc) then start running the tool to update projects - work with @priya91 for any help needed here;  lastly, please think about what test cases to add to validate intellisense comments (smart quotes, missing comments on new classes, etc).  Thanks